packages/cozy-procedures/src/containers/DocumentsDataForm.jsx

```diff
@@ -1,13 +1,18 @@
 import { connect } from 'react-redux'
 import context from '../redux/context'
-import { fetchDocument, getDocuments } from '../redux/documentsDataSlice'
+import {
+  fetchDocument,
+  getDocuments,
+  unlinkDocument
+} from '../redux/documentsDataSlice'
 
 const mapStateToProps = state => ({
   data: getDocuments(state)
 })
 
 const mapDispatchToProps = {
-  fetchDocument
+  fetchDocument,
+  unlinkDocument
```

Comment:
That does mean that `DocumentHolder` also receives `fetchDocument` and the list of documents, right?

Reply:
Yep. Is it problematic for you?

Reply:
On principle yes, the component will get props that it doesn't use or want. But in practice I don't mind, it can stay as it is for me :)